.github/workflows/cicd-pipeline.yml

```
@@ -316,23 +311,59 @@ jobs:
-          git fetch origin develop
-          git checkout develop
-          git merge origin/main --no-ff -m "chore: sync main after release [skip ci]"
-          git push origin develop
-          echo "Develop branch is now up-to-date with main."
+        id: create_sync_branch
+        uses: actions/checkout@v5
+        with:
+          ref: 'develop' # Começa da develop
+          fetch-depth: 0
+          ssh-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - run: |
```

actionlint (1.7.8)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)